<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="41802" started-at="2020-10-28T18:29:19Z" finished-at="2020-10-28T18:30:01Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="41802" started-at="2020-10-28T18:29:19Z" finished-at="2020-10-28T18:30:01Z">
      <class name="com.ebanking.testcases.TC_addCustomertest">
        <test-method status="PASS" signature="setUp(java.lang.String)[pri:0, instance:com.ebanking.testcases.TC_addCustomertest@6c7a164b]" name="setUp" is-config="true" duration-ms="9341" started-at="2020-10-28T23:59:19Z" finished-at="2020-10-28T23:59:29Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="setUp(java.lang.String)[pri:0, instance:com.ebanking.testcases.TC_loginTest_001@222a59e6]" name="setUp" is-config="true" duration-ms="10960" started-at="2020-10-28T23:59:29Z" finished-at="2020-10-28T23:59:40Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="addNewCustomer()[pri:0, instance:com.ebanking.testcases.TC_addCustomertest@6c7a164b]" name="addNewCustomer" duration-ms="17883" started-at="2020-10-28T23:59:43Z" finished-at="2020-10-29T00:00:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addNewCustomer -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.ebanking.testcases.TC_addCustomertest@6c7a164b]" name="tearDown" is-config="true" duration-ms="50" started-at="2020-10-29T00:00:01Z" finished-at="2020-10-29T00:00:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.ebanking.testcases.TC_loginTest_001@222a59e6]" name="tearDown" is-config="true" duration-ms="105" started-at="2020-10-29T00:00:01Z" finished-at="2020-10-29T00:00:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.ebanking.testcases.TC_addCustomertest -->
      <class name="com.ebanking.testcases.TC_loginTest_001">
        <test-method status="FAIL" signature="loginTest()[pri:0, instance:com.ebanking.testcases.TC_loginTest_001@222a59e6]" name="loginTest" duration-ms="3085" started-at="2020-10-28T23:59:40Z" finished-at="2020-10-28T23:59:43Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at org.testng.Assert.assertTrue(Assert.java:51)
	at com.ebanking.testcases.TC_loginTest_001.loginTest(TC_loginTest_001.java:67)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
      </class> <!-- com.ebanking.testcases.TC_loginTest_001 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
